src/Entity/FrenchBulldog.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+final class FrenchBulldog extends AnimalAbstract implements AnimalInterface
+{
+    public function animalDescription(): string
+    {
+        return 'French Bulldog';
+    }
+
+    public function animalPhoto(): string
+    {
+        return 'images/frenchbulldog.jpg';
+    }
+
+    public function loadAnimal()
+    {
+        return 'Papaia - '. $this->animalDescription() . '<br><img width="500px" src="'.$this->animalPhoto().'">' . '<br><br>';
```

Comment:
Shouldn't this be:

```suggestion
        return 'Papaia - '. parent::loadAnimal();
```
?

Reply:
This is to show that if you want to be specific you can also have a loadAnimal in the entity with different implementation. In this case I just added the name of the dog.

Reply:
Yes I see, but did you want to override it completely or just adding the dogs name?

Reply:
In this case is to show the option to overwrite, I just added the name as an example but it could be completely change the html or something like that.